src/packagedcode/pyrpm/rpm.py

```diff
@@ -1,8 +1,8 @@
-# -*- coding: iso-8859-15 -*-
+# -*- coding: utf-8 -*-
```

Comment:
Again this is a repetition of https://github.com/nexB/scancode-toolkit/pull/1534/commits/31dd17cd3e3d83db44a545264b12503572780936 .